templates/utilities.html

```diff
@@ -5,16 +5,30 @@
 <div class="col-sm-6">
 <form action="/utility">
     <img class="mb-4" src="https://zap.me/app/themes/zap/assets/public/images/logo-footer.svg" alt="" width="72" height="72">
-    <h1 class="h3 mb-3 font-weight-normal">Please choose a utility</h1>
-    <div class="form-group">
-        <label for="utility">Utilities:</label>
-        <select name="utility" id="utility" class="form-control">
-            {% for utility in utilities %}
-            <option value="{{utility.id}}">{{utility.name}}</option>
-            {% endfor %}
-        </select>
+    {% if kyc_info['level'] < '2' %}
```

Comment:
remove the oauth login/logout stuff from this template and into the layout.html (use a navbar: https://getbootstrap.com/docs/4.0/components/navbar/)